studio/schemas/documents/compensations.ts

```diff
@@ -28,14 +29,23 @@ const compensations = defineType({
       ...titleSlug,
       type: "internationalizedArrayString",
     },
+    // defineField({
+    //   name: "showSalaryCalculator",
+    //   title: "Show Salary Calculator",
+    //   description:
+    //     "Toggle this setting to show or hide the salary calculator on the page. The salary calculator helps users estimate their total compensation based on various factors such as salary data, pension and bonuses.",
+    //   type: "boolean",
+    //   initialValue: true,
+    // }),
```

Comment:
We might just delete this toggleswitch!

Reply:
I contemplated this (and usually a fan of deleting instead of commenting out), but I was thinking this really was temporary buuuuuut probably not